lib/ld-key-pair.js

```diff
@@ -352,6 +354,34 @@ class RSAKeyPair extends LDKeyPair {
     return keys;
   }
 
+  validateKeyParams() {
+    if(this.publicKeyPem) {
+      const publicKey = forge.pki.publicKeyFromPem(this.publicKeyPem);
+      const keyBits = publicKey.n.bitLength();
+      if(keyBits !== DEFAULT_RSA_KEY_BITS) {
+        throw new Error(`Invalid RSA keyBit length ${JSON.stringify(keyBits)}` +
+          ` required value is ${DEFAULT_RSA_KEY_BITS}`);
+      }
+      if(publicKey.e.toString(16) !== '10001') {
+        throw new Error(`Invalid RSA exponent ${JSON.stringify(publicKey.e.toString(16))}` +
+          ` required value is 10001}`);
```

Comment:
Should probably say the value is 65537 here since 10001 is in hex and may confuse readers.

Reply:
Could to `toString(10)` and compare against that for clarity.

Reply:
Ah yeah, good call. Changed.